resources/spec.yaml

```diff
@@ -4492,6 +4492,323 @@ paths:
+      security:
+        - api_token: []
+      summary: Get org settings
+      tags:
+        - Org
+    post:
+      consumes:
+        - application/json
+      description: Partial update settings of your organization.
+      operationId: partialSettingUpdate
+      parameters:
+        - in: body
+          name: orgSetting
+          required: true
+          schema:
+            $ref: '#/definitions/OrgSetting'
+      responses:
+        '200':
+          description: Success
+          schema:
+            $ref: '#/definitions/OrgSetting'
+      security:
+        - api_token: []
+      summary: Partial settings update
+      tags:
+        - Org
+    put:
+      consumes:
+        - application/json
+      description: Update settings of your organization.
+      operationId: settingUpdate
+      parameters:
+        - in: body
+          name: orgSetting
+          required: true
+          schema:
+            $ref: '#/definitions/OrgSetting'
+      responses:
+        '200':
+          description: Success
+          schema:
+            $ref: '#/definitions/OrgSetting'
+      security:
+        - api_token: []
+      summary: Settings update
+      tags:
+        - Org
+  /api/v1/org/contacts:
+    get:
+      consumes:
+        - application/json
+      description: Gets Contact Types of your organization.
+      operationId: getOrgContactTypes
+      parameters: []
+      responses:
+        '200':
+          description: Success
+          schema:
+            items:
+              $ref: '#/definitions/OrgContactTypeObj'
+            type: array
+      security:
+        - api_token: []
+      summary: Get org contact types
+      tags:
+        - Org
+  '/api/v1/org/contacts/{contactType}':
+    get:
+      consumes:
+        - application/json
+      description: Retrieves the URL of the User associated with the specified Contact Type.
+      operationId: getOrgContactUser
+      parameters:
+        - in: path
+          name: contactType
+          required: true
+          type: string
+      responses:
+        '200':
+          description: Success
+          schema:
+            $ref: '#/definitions/OrgContactUser'
+      security:
+        - api_token: []
+      summary: Get org contact user
+      tags:
+        - Org
+    put:
+      consumes:
+        - application/json
+      description: Updates the User associated with the specified Contact Type.
+      operationId: updateOrgContactUser
+      parameters:
+        - in: path
+          name: contactType
+          required: true
+          type: string
+        - in: body
+          name: userId
+          required: true
+          schema:
+            $ref: '#/definitions/UserIdString'
+      responses:
+        '200':
+          description: Success
+          schema:
+            $ref: '#/definitions/OrgContactUser'
+      security:
+        - api_token: []
+      summary: Update org contact user
+      tags:
+        - Org
+  /api/v1/org/logo:
```

Comment:
Why is this left here? I thought we're going to add it as part of another [task](https://oktainc.atlassian.net/browse/OKTA-424553)?

Reply:
Removed